Sources/protoc-gen-swift/Descriptor+Extensions.swift

```diff
@@ -352,13 +352,15 @@ extension FieldDescriptor {
         case "inf": return "Double.infinity"
         case "-inf": return "-Double.infinity"
         case "nan": return "Double.nan"
+        case "-nan": return "Double.nan"
```

Comment:
Should this return "-Double.nan"?

Reply:
There really isn't such a thing as negative nan, it's something the parsing is supposed to allow.

Reply:
Question opened up with the protobuf-team about the tracking of the sign.